packages/headless/src/features/configuration/search-configuration-actions-loader.ts

```diff
@@ -30,7 +30,7 @@ export interface SearchConfigurationActionCreators {
  * @returns An object holding the action creators.
  */
 export function loadSearchConfigurationActions(
-  engine: SearchEngine
+  engine: CoreEngine
```

Comment:
Is this right ? What problem does this fix ?

Reply:
If it only works for those two engines, then perhaps `SearchEngine | RecommendationEngine` would be better.